package.json

```diff
@@ -21,7 +21,9 @@
     "build-mac": "build --mac --publish never",
     "build-linux": "build --linux --publish never",
     "build-win": "build --win --publish never",
-    "test": "xvfb-maybe mocha test --full-stack"
+    "test": "xvfb-maybe mocha test --full-stack",
+    "markdown-release-notes": "node ./src/about/modify.js '../../CHANGELOG.md' 'release_notes.html'",
+    "markdown-package-details": "node ./src/about/modify.js '../../PACKAGE_DETAILS.md' 'package_details.html'"
```

Comment:
If the html files are generated from "source", should they also be checked in to the repo?

Reply:
Thank you for the reminder - I need to update the expeditor script to check in those modified documents

Reply:
Also, I have them checked in right now because the update script only modifies _part_ of the HTML document. Otherwise I would need to check in the JS equivalent of a `.erb` and use template filling to generate them

Reply:
Ooooh, I see. OK. 👍